### ISSUE TITLE
IrisMobile gets stuck on "Checking APK Integrity"

### ISSUE THREAD

**Issue by bfabe8** [ScarletStudy/DGS1-Android-Release#52]
### About your device

**Device Manufacturer / Model or Emulator / Version??**
Nox 6.2.8.3

**Android Version?**
7.1.2

**Are you using a Custom ROM?** (If you don't know what this means, then most likely no)
No

**Is your device rooted?** (If you don't know what this means, then most likely no)
No

### Your issue

I've downloaded DGS from the google play store onto my Nox installation, and IrisMobile detects the installation - however, when it reaches "Checking APK Integrity" it makes no progress and simply stalls. I've tried reinstalling both IrisMobile and DGS, and clearing the ScarletStudy/IrisMobile folder, all to no avail.

**Comment by wabberz** [comment]
Sorry, this issue has been overlooked. Were you able to solve this?

**Comment by bfabe8** [comment]
No, I wasn't. I had to switch to using Bluestacks which worked perfectly.
